net/ddns-go/file/ddns-go.init

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2021-2022  sirpdboy  <herboy2008@gmail.com> https://github.com/sirpdboy/luci-app-ddns-go 
+#
+# This file is part of ddns-go .
+# 
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+
+START=99
+USE_PROCD=1
+
+PROG=/usr/bin/ddns-go
+CONFDIR=/etc/ddns-go
+CONF=/etc/ddns-go/ddns-go-config.yaml
```

Comment:
Perhaps the package should have a `conffiles` section to save this file / directory.